pybamm/plotting/quick_plot.py

```diff
@@ -464,6 +464,8 @@ def plot(self, t, dynamic=False):
         ----------
         t : float
             Dimensional time (in 'time_units') at which to plot.
+        dynamic : bool
+
```

Comment:
You could include a short description, e.g. "Whether the plot should be dynamic or not"

Reply:
It is also optional

Reply:
added a short description, as well as the optional tag.